From: Norbert Preining <norbert@preining.info>
Date: Sun, 29 Dec 2019 13:31:54 +0900
Subject: allow for plugin update check, but no calibre version check

Forwarded: not-needed

allow for plugin update check, but no calibre version check
---
 src/calibre/gui2/update.py | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/calibre/gui2/update.py b/src/calibre/gui2/update.py
index 61194cf..d9c24b7 100644
--- a/src/calibre/gui2/update.py
+++ b/src/calibre/gui2/update.py
@@ -29,6 +29,7 @@ def get_download_url():
 
 
 def get_newest_version():
+    return NO_CALIBRE_UPDATE
     icon_theme_name = icon_resource_manager.user_theme_name
     if icon_theme_name == 'default':
         icon_theme_name = ''
